.tekton/pull-request.yaml

```diff
@@ -44,6 +44,66 @@ spec:
         params:
         - name: args
           value: ["."]
+      - name: build-container
+        runAfter:
+          - yaml-lint-check
+        params:
+          - name: IMAGE
+            value: quay.io/redhat-appstudio/pull-request-builds:build-definitions-{{revision}}
+          - name: CONTEXT
+            value: appstudio-utils
+        taskRef:
+          name: buildah
+        workspaces:
+          - name: source
+            workspace: workspace
+      - name: build-bundles
+        params:
+          - name: revision
+            value: "{{ revision }}"
+        runAfter:
+          - build-container
+        workspaces:
+          - name: source
+            workspace: workspace
+        taskSpec:
+          params:
+            - name: revision
+              type: string
+          steps:
+            - name: build-bundles
+              image: quay.io/redhat-appstudio/pull-request-builds:build-definitions-{{revision}}
+              workingDir: $(workspaces.source.path)
+              script: |
+                #!/usr/bin/env bash
+                MY_QUAY_USER=redhat-appstudio TEST_REPO_NAME=pull-request-builds BUILD_TAG=$(params.revision) SKIP_BUILD=1 hack/build-and-push.sh
+          workspaces:
+            - name: source
+      - name: e2e-test
+        runAfter:
+          - build-bundles
+        taskSpec:
+          steps:
+            - name: e2e-test
+              image: quay.io/psturc/e2e-tests:PLNSRVCE-149
```

Comment:
https://github.com/redhat-appstudio/e2e-tests/pull/47

Reply:
TODO: update this to `quay.io/redhat-appstudio/e2e-tests:latest` once the PR ^ gets merged